### PR TITLE
Release 1.4.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
@ekohl should we ignore the psych error for now or pin it somehow?

**Comment by ekohl** [comment]
We may need to pin rdoc if we pull that in somehow. But then to get rid of it, we need to invalidate the cache, which is annoying. I'd be OK with ignoring it for now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+[Full Changelog](https://github.com/voxpupuli/puppet_metadata/compare/1.3.0...1.4.0)
+
+**Implemented enhancements:**
+
+- Add Arch Linux support [\#37](https://github.com/voxpupuli/puppet_metadata/pull/37) ([ekohl](https://github.com/ekohl))
+
+**Closed issues:**
+
+- fails to parse metadata with issues\_url set to null [\#31](https://github.com/voxpupuli/puppet_metadata/issues/31)
+- Hash matrix for Github Actions makes excludes impossible [\#13](https://github.com/voxpupuli/puppet_metadata/issues/13)
```
Comment (ekohl) [inline CHANGELOG.md:14]: We didn't really resolve these issues in this release, did we? Should we skip them?

Comment (bastelfreak) [reply CHANGELOG.md]: updated it

Comment (ekohl) [reply CHANGELOG.md]: I actually meant both.

**Review by ekohl — APPROVED** [review]

**Pull request merged**